package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "to-do-list-project",
+  "version": "1.0.0",
+  "description": "<div align=\"center\"> <a id=\"readme-top\"></a>",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.27.5",
+    "hint": "^2.0.0",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
+  }
+}
```

Comment:
- [ ] **[OPTIONAL]**  Currently in order to start your project we have to navigate from the root directory to ```/toDoList``` folder, Kindly move all the subfolders of **toDoList** to the root directory to make the project's main files easier to navigate. 🙏